nemo/collections/nlp/modules/common/megatron/megatron_init.py

```diff
@@ -95,6 +95,8 @@ def initialize_model_parallel_for_nemo(
     virtual_pipeline_model_parallel_size=None,
     pipeline_model_parallel_split_rank=None,
     context_parallel_size=1,
+    encoder_tensor_model_parallel_size=0,
+    encoder_pipeline_model_parallel_size=0,
```

Comment:
Should the default be `0` or `None`?

Reply:
0 following mcore here https://github.com/NVIDIA/Megatron-LM/blob/98abe37866bba8aa0eee246fdac5163f5c8bcff7/megatron/core/parallel_state.py#L354